app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,18 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 skill_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._skill_level = skill_level
+
+    def player_info(self) -> str:
+        name = self.nickname
+        lvl = self._skill_level
+        return f"Dwarf blacksmith {name} with skill of the {lvl} level"
```

Comment:
```suggestion
        return f"Dwarf blacksmith {self.nickname} with skill of the {self._skill_level} level"
```

Reply:
I think it's better, but you don't have to change

Reply:
btw, agree